server/views/pages/accessibilityStatement/index.njk

```diff
@@ -0,0 +1,58 @@
+{% extends "../../partials/layout.njk" %}
+
+{% set pageTitle = "Accessibility statement" %}
+
+{% set pageId = "accessibility statement" %}
+
+{% block beforeContent %}
+  <nav class="govuk-breadcrumbs govuk-!-display-none-print" aria-label="Breadcrumb">
```

Comment:
I don't think we want the breadcrumbs on this page.
Given what I wrote above re: open access to this page, where one of the breadcrumb links is to the PLP Prisoner List page, that clearly wont work if the user in question is not a PLP user.